portfolio/pom.xml

```diff
@@ -23,11 +23,18 @@
       <version>4.0.1</version>
```

Comment:
A more general comment is to add more information in the description of this pull request. Basically, what you're changing, refactoring, or implementing, and any TODOs remaining.